habitat-lab/habitat/config/CONFIG_KEYS.md

```diff
@@ -0,0 +1,180 @@
+|rgb_head_agent| An Agent with a single RGB camera attached to its head.|
+|rgbd_head_rgbd_arm_agent| An Agent with both a RGB and depth camera attached to its head and arm.|
+|depth_head_agent_vis| An Agent with a depth camera attached to its head and a third person RGB camera on top of the robot.|
+
+
+
+
+## Environment
+Some habitat environment configurations.
+| Key | Description |
+| --- | --- |
+|habitat.environment.max_episode_steps| The maximum number of environment steps before the episode ends.|
+|habitat.environment.max_episode_seconds| The maximum number of seconds steps before the episode ends.|
+
+## Navigation Actions
+The way one would add an action to a configuration file would be by adding to the `defaults` list. For example:
+```
+defaults:
+  - /habitat/task/actions:
+    - move_forward
+    - turn_left
+```
+
+| Key | Description |
+| --- | --- |
+| habitat.task.actions.stop |     In Navigation tasks only, the stop action is a discrete action. When called, the Agent will request to stop the navigation task. Note that this action is needed to succeed in a Navigation task since the Success is determined by the Agent calling the stop action within range of the target. Note that this is different from the RearrangeStopActionConfig that works for Rearrangement tasks only instead of the Navigation tasks.|
+| habitat.task.actions.empty | In Navigation tasks only, the pass action. The robot will do nothing.|
+| habitat.task.actions.move_forward |     In Navigation tasks only, this discrete action will move the robot forward by a fixed amount determined by the `habitat.simulator.forward_step_size` amount. |
+| habitat.task.actions.turn_left |     In Navigation tasks only, this discrete action will rotate the robot to the left  by a fixed amount determined by the `habitat.simulator.turn_angle` amount. |
+| habitat.task.actions.turn_right |     In Navigation tasks only, this discrete action will rotate the robot to the right by a fixed amount determined by the `habitat.simulator.turn_angle` amount. |
+| habitat.task.actions.look_up |      In Navigation tasks only, this discrete action will rotate the robot's camera up by a fixed amount determined by the `habitat.simulator.tilt_angle` amount. |
+| habitat.task.actions.look_down |      In Navigation tasks only, this discrete action will rotate the robot's camera down by a fixed amount determined by the `habitat.simulator.tilt_angle` amount. |
+
+## Navigation Measures
+The way one would add an action to a configuration file would be by adding to the `defaults` list. For example:
+```
+defaults:
+  - /habitat/task/measurements:
+    - robot_force
+    - force_terminate
+```
+
+| Key | Description |
+| --- | --- |
+| habitat.task.measurements.num_steps| In both Navigation and Rearrangement tasks, counts the number of steps since  the start of the episode.|
+| habitat.task.measurements.distance_to_goal | In Navigation tasks only, measures the geodesic distance to the goal.|
```

Comment:
why only in navigation? Can't we use this in rearrangement, for example, for distance to object?

Reply:
Nope. Different measure.